packages/forge/src/governance/extensions/GovernorSorting.sol

```diff
@@ -0,0 +1,229 @@
+
+    /**
+     * @dev Getter if a given ranking is tied.
+     */
+    function isTied(uint256 ranking) public view returns (bool) {
+        require(
+            setSortedAndTiedProposalsHasBeenRun, "RewardsModule: run setSortedAndTiedProposals() to populate this value"
+        );
+        return _isTied[ranking];
+    }
+
+    /**
+     * @dev Getter for tiedAdjustedRankingPosition of a ranking.
+     */
+    function rankingPosition(uint256 ranking) public view returns (uint256) {
+        require(
+            setSortedAndTiedProposalsHasBeenRun, "RewardsModule: run setSortedAndTiedProposals() to populate this value"
+        );
+        return tiedAdjustedRankingPosition[ranking];
+    }
+
+    /**
+     * @dev Getter for _sortedProposalIds.
+     */
+    function sortedProposalIds() public view returns (uint256[] memory) {
+        require(
+            setSortedAndTiedProposalsHasBeenRun, "RewardsModule: run setSortedAndTiedProposals() to populate this value"
+        );
+        return (_sortedProposalIds);
+    }
+
+    /**
+     * @dev Getter for the lowest ranking.
+     */
+    function lowestRanking() public view returns (uint256) {
+        require(
+            setSortedAndTiedProposalsHasBeenRun, "RewardsModule: run setSortedAndTiedProposals() to populate this value"
+        );
+        return _lowestRanking;
+    }
+
+    /**
+     * @dev Getter for highest tied ranking.
+     */
+    function highestTiedRanking() public view returns (uint256) {
+        require(
+            setSortedAndTiedProposalsHasBeenRun, "RewardsModule: run setSortedAndTiedProposals() to populate this value"
+        );
+        return _highestTiedRanking;
+    }
+
+    /**
+     * @dev Accessor to the internal vote counts for a given proposal.
+     */
+    function allProposalTotalVotes()
+        public
+        view
+        virtual
+        returns (uint256[] memory proposalIdsReturn, VoteCounts[] memory proposalVoteCountsArrayReturn)
+    {
+        uint256[] memory proposalIds = getAllProposalIds();
+        VoteCounts[] memory proposalVoteCountsArray = new VoteCounts[](proposalIds.length);
+        for (uint256 i = 0; i < proposalIds.length; i++) {
+            proposalVoteCountsArray[i] = proposalVotesStructs[proposalIds[i]].proposalVoteCounts;
+        }
+        return (proposalIds, proposalVoteCountsArray);
+    }
+
+    /**
+     * @dev Accessor to the internal vote counts for a given proposal that excludes deleted proposals.
+     */
+    function allProposalTotalVotesWithoutDeleted()
+        public
+        view
+        virtual
+        returns (uint256[] memory proposalIdsReturn, VoteCounts[] memory proposalVoteCountsArrayReturn)
+    {
+        uint256[] memory proposalIds = getAllProposalIds();
+        uint256[] memory proposalIdsWithoutDeleted = new uint256[](proposalIds.length);
+        VoteCounts[] memory proposalVoteCountsArray = new VoteCounts[](proposalIds.length);
+
+        uint256 newArraysIndexCounter = 0;
```

Comment:
No, because `i` is the index of the original array which includes deleted proposals and `newArraysIndexCounter` is the index of the new array which doesn't.

Reply:
Oh yes, newArraysIndexCounter is incremented only conditionally, my bad!

Reply:
No worries, thanks for checking!!